src/libraries/System.Numerics.Vectors/tests/Matrix4x4Tests.cs

```diff
@@ -2152,7 +2215,11 @@ private void CreateBillboardFact(Vector3 placeDirection, Vector3 cameraUpVector,
         public void Matrix4x4CreateBillboardTest01()
         {
             // Object placed at Forward of camera. result must be same as 180 degrees rotate along y-axis.
-            CreateBillboardFact(new Vector3(0, 0, -1), new Vector3(0, 1, 0), Matrix4x4.CreateRotationY(MathHelper.ToRadians(180.0f)));
+            CreateBillboardFact(
+                new Vector3(0, 0, -1),
+                new Vector3(0, 1, 0),
```

Comment:
nit: You can use `Vector3.UnitY` and such for cases like this, where only a single axis is `1`